components/helper/ConfirmModal.tsx

```diff
@@ -0,0 +1,83 @@
+import { Modal, Box, Typography, Button, createTheme, ThemeProvider }
+  from '@mui/material';
+import { grey } from '@mui/material/colors';
+
+function ConfirmModal({
+  open,
+  onClose,
+  message,
+  onConfirm,
+  onCancel,
+  confirmButtonName,
+  colorButton
+}) {
+
+  const theme = createTheme({
+    components: {
+      MuiButton: {
+        styleOverrides: {
```

Comment:
is there a way to avoid specifying the style?

Reply:
can i get a hint ?